### ISSUE TITLE
"Push to mute" just like "Push to talk" just for muting

### ISSUE THREAD

**Issue by chagai95** [jitsi/jitsi-meet#9130]
**Is your feature request related to a problem you are facing?**
When I sneeze I can't manage to find the mouse/click the m before everyone hears it 

**Describe the solution you'd like**
Push the space button to mute until I let go

**Describe alternatives you've considered**
I usually use my headset mute mic button.

Thanks to @gjaekel

**Comment by damencho** [comment]
This is already the case with push to talk. If you click it once and release it will mute you ... 
But this is going to change when this lands https://github.com/jitsi/jitsi-meet/pull/8921 as push to talk will start working only when muted (as the name says)

**Comment by gjaekel** [comment]
No; it's meant to act **exactly** in the other way round as "Push-To-Talk" during muted state. If not in muted state, the space bar should interrupt the audio as long as pressed: "Push-To-Mute"

**Comment by damencho** [comment]
We had recently changed the behavior of push to talk, and we needed to revert it by explicit request from Product, as this is a privacy issue.
The case covered is exactly as the problem you describe.
If you want to make sure you are muted just touch the space bar. If you will sneeze space is the biggest key on the keyboard to help you mute - just press it once and you are good to go :)
There are people while entering a conference always press the space bar to make sure they are muted.
